origami-lib/test/test_textimageinput.py

```diff
@@ -28,6 +28,20 @@ def test_getTextInput():
     assert(len(text_array) == 3)
     assert(text_array == ['hello', 'world', 'Is this a sample question?'])
 
+# send text strings
+def test_getTextInput2():
```

Comment:
Thanks for writing this but this seems to be same as the previous test case. Can you please try writing tests for the corner cases. For example, if the inputs are empty then what will be output and so on.